meteor/client/ui/RundownView/RundownTiming/SegmentDuration.tsx

```diff
@@ -9,6 +9,8 @@ interface ISegmentDurationProps {
 	label?: ReactNode
 	/** If set, the timer will display just the played out duration */
 	countUp?: boolean
+	/** Always show planned segment duration instead of counting up/down */
+	static?: boolean
```

Comment:
`static` is a keyword https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static this should be renamed to something else

Reply:
Renamed to `fixed` and `fixedSegmentDuration`